tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "outDir": "./dist/",
     "noImplicitAny": true,
-    "module": "es6",
+    "module": "esnext",
```

Comment:
Could this have some implications on browser compatibility? What doesn't work with `es6`?

Reply:
I have no clue :shrug:. If you change it back, eslint's gonna rant about our current lazy imports not being possible. So I guess we were not compatible before ? But without knowing it.